res/games/gameitem.css

```diff
@@ -0,0 +1,117 @@
+{%- macro playermacro(player,teamalign) %}
+        {% if player.login is not defined %}
+            {% set player_login = player %}
+        {% else %}
+            {% set player_login = player.login %}
+        {% endif %}
+        
+        {% set player_country = "" %}
+        {% set player_global_rating = "???" %}
+        {% if player.countrypath is defined %}
+            {% set player_country = player.countrypath %}
+            {% set player_global_rating = (player.global_rating[0]-(player.global_rating[1]*3))|int %}
+        {% endif %}
+
+        {% set width = 130 %}
+        {% if teams[1] is not defined %}
+            {% set alignment = 'left' %}
+            {% if "-" not in player.login %}
```

Comment:
because for some reason, when a player has "-" in his name, HTML will make a new line after each "-"... and that looks ugly as hell :/

Reply:
We should find the root cause of that instead of special-casing it. Maybe we're generating invalid html somewhere or not html-escaping things?

Reply:
i could not find anything to that - but it may not be directly related to HTML after all. and it does not *always* make a new line, but:

when you dont set the width of the column manually, in some cases it will generate a new line when the string is really long. i prevented that by making the column more wide, but when there is only one team, i set the width to 0 - that forces the box to be as small as possible.

and heres the catch: this works with even very long names. no new line created. ONLY when there is a `-` in the name...

Reply:
Quick googling suggests this, for example:
https://stackoverflow.com/questions/1904612/stop-wrapping-of-hyphenated-text
It might be that there's word wrap enabled by default. Try setting a no-wrap style.

Reply:
yes, i tried that - it works with normal long names, but still, as soon as there are "-" (now i know theyre called hyphen xD) involved, theres a new line :(